seatunnel-e2e/seatunnel-connector-v2-e2e/connector-paimon-e2e/src/test/resources/fake_to_paimon.conf

```diff
@@ -29,7 +29,7 @@ env {
 
 source {
   FakeSource {
-    row.num = 100000
```

Comment:
Why change it?

Reply:
@SimonChou12138 @TyrantLucifer 
- #5374  

To reproduce failure, #5419 set `row.num` with a large number. It does not seem affect test validation that we set `row.num=10000` here.

Reply:
I think the row.num = 100000 here should not affect the passage of the PR. I will have a closer look at the GC problem in the future.

Reply:
You can config FakeSource `split.num` to solve the OOM.

Reply:
@CheneyYin Copy that. I'll try later.Thanks,bro.